lineman/app/components/thread_page/comment_form/comment_form.coffee

```diff
@@ -1,35 +1,40 @@
 angular.module('loomioApp').directive 'commentForm', ->
-  scope: {comment: '='}
+  scope: {discussion: '='}
```

Comment:
I always felt it was kinda weird we were passing a comment to the comment form, then pulling the discussion and group out of it. Now we're not doing that anymore!